INFO     self_developed_model.conftest:conftest.py:62 [2025-10-09 14:05:29] 用例 test_cloudTrader/test_delete.py::TestDelete_cloudTrader::test_delete_cloudMaster_hand 开始登录，环境: uat
INFO     self_developed_model.conftest:conftest.py:100 UAT登录尝试 1/5，MFA验证码: 355777
DEBUG    urllib3.connectionpool:connectionpool.py:1051 Starting new HTTPS connection (1): uat.atcp.top:443
DEBUG    urllib3.connectionpool:connectionpool.py:546 https://uat.atcp.top:443 "POST /api/sys/auth/login HTTP/11" 200 None
INFO     self_developed_model.conftest:conftest.py:124 UAT登录成功（第1次尝试）
INFO     self_developed_model.commons.variable_manager:variable_manager.py:111 [2025-10-09 14:03] 运行时变量已保存到: D:\pycharm_test\self_developed_model\VAR\runtime_vars_cloud.json
DEBUG    urllib3.connectionpool:connectionpool.py:546 https://uat.atcp.top:443 "DELETE /api/mascontrol/cloudMaster HTTP/11" 200 None
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-10-09 14:05:30] DELETE请求: /mascontrol/cloudMaster | 数据: [269]
INFO     self_developed_model.conftest:conftest.py:157 [2025-10-09 14:05:31] 用例 test_cloudTrader/test_delete.py::TestDelete_cloudTrader::test_delete_cloudMaster_hand 执行完毕